README.md

```diff
@@ -134,12 +134,12 @@ List all migrations with an indicator (applied,pending) whether is has been appl
 
 Run this command in the directory where all migrations are stored. Use `--migrations` for a different location.
 
-### plan  [path] [|migration file] [--migrations folder]
```

Comment:
the pipe was included to hint that the migration file argument is not needed

Reply:
Then sorry, my bad :)

Reply:
I propose to change the documentation and code to match the following convention ( https://developers.google.com/style/code-syntax )

- mandatory argument just as it is
- voluntary argument in square brackets ==> [voluntary]
- mandatory argument with mutually exclusive options in curly brackets separated by a pipe ==> {one|two|three}
- voluntary argument with mutually exclusive options in square brackets separated by a pipe ==> [one|two|three]

Reply:
yes, let's approve it like this